paimon-spark/paimon-spark-2/pom.xml

```diff
@@ -84,6 +89,11 @@ under the License.
             <version>${jackson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.xbean</groupId>
+            <artifactId>xbean-asm6-shaded</artifactId>
+            <version>${asm6.version}</version>
```

Comment:
which scope? test or provided?

Reply:
@JingsongLi  This package is to replace the package in `spark-sql_2.11`, so keep the scope same with `spark-sql_2.11`, namely default `compile`